recipes/libcublas/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Install to conda style directories
+[[ -d lib64 ]] && mv lib64 lib
+mkdir -p ${PREFIX}/lib
+[[ -d pkg-config ]] && mv pkg-config ${PREFIX}/lib/pkgconfig
+[[ -d "$PREFIX/lib/pkgconfig" ]] && sed -E -i "s|cudaroot=.+|cudaroot=$PREFIX|g" $PREFIX/lib/pkgconfig/cublas*.pc
```

Comment:
This is getting written as:
```
cudaroot=/home/conda/staged-recipes/build_artifacts/libcublas_1680041891887/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p
```

Is that what we want?

Reply:
This is actually fine. 😄  I looked into how this works for other libraries. There is a key in `info/paths.json` that contains a `prefix_placeholder` that is used to replace this at installation time.

```json
{
  "paths": [
    // ...
    {
      "_path": "lib/pkgconfig/cublas-12.0.pc",
      "file_mode": "text",
      "path_type": "hardlink",
      "prefix_placeholder": "/home/conda/staged-recipes/build_artifacts/libcublas_1680041891887/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_p",
      "sha256": "7a4104c5245abff556a98edd6e4a467e43030992aa35a7ad2b727df9741f70d7",
      "size_in_bytes": 491
    },
    // ...
  ],
  "paths_version": 1
}
```

Reply:
Yeah that looks right.

Conda-build will [identify the path (since it is absolute) and mark it for replacement]( https://docs.conda.io/projects/conda-build/en/latest/resources/make-relocatable.html ) (for [example]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=587369&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=568 )). This will get marked in the Conda package's `has_prefix` file.

Whenever Conda installs the package, it will know that these `*.pc` files are candidates for prefix replacement using the environment path.